FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/Demos.kt

```diff
@@ -161,25 +161,25 @@ val V1DEMO = arrayListOf(
 )
 
 val V2DEMO = arrayListOf(
-    Demo(V2APP_BAR_LAYOUT, V2AppBarActivity::class, Badge.Modified),
```

Comment:
I think we can keep the badge for someting, because there were a lot of changes in the previous release